allium/lib/network_health.py

```
@@ -340,23 +421,39 @@ def _integrate_aroi_validation(health_metrics, relay_set, total_relays_count):
             health_metrics['avg_relays_per_aroi_operator'] = 0.0
                 
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)